internal/cmd/egctl/translate_test.go

```diff
@@ -185,6 +185,7 @@ func TestTranslate(t *testing.T) {
 			output:       yamlOutput,
 			resourceType: string(AllEnvoyConfigType),
 			expect:       true,
+			extraArgs:    []string{"--add-missing-resources"},
```

Comment:
this is needed for tests to pass 🤷

Reply:
ummm have you tried running `make testdata` , in auto generates the golden testdata 😄